modules/core/src/main/game/light.scala

```diff
@@ -12,7 +13,8 @@ case class LightGame(
     whitePlayer: LightPlayer,
     blackPlayer: LightPlayer,
     status: Status,
-    win: Option[Color]
+    win: Option[Color],
+    variant: Variant
```

Comment:
make LightGame less light. It's worth noting it as there are millions of instances flowing through lila.